web/src/hash_parser.ts

```diff
@@ -15,6 +15,18 @@ export function get_hash_section(hash?: string): string {
     return parts[1] || "";
 }
 
+export function get_current_nth_hash_section(n: number): string {
```

Comment:
I chose to do 1-indexing instead of 0-indexing, but happy to change it if 0 feels more intuitive to other folks

Reply:
This is probably mostly fine for now